### ISSUE TITLE
empty first field leads to "Note creation failed for unknown reason"

### ISSUE THREAD

**Issue by 517qf** [louietan/anki-editor#18]
I use the addon [Add Note ID](https://ankiweb.net/shared/info/1672832404). This is useful in certain cases if you export and import a deck so that for these cases it's recommended in the manul, see https://apps.ankiweb.net/docs/manual.html#exporting-text.

When I push a note from emacs to Anki I don't fill out the field "Note ID". The problem is that on some of my notes the field "Note ID" is the first field. If I have an empty first field I get an error ":ANKI_FAILURE_REASON: Note creation failed for unknown reason".

Having the first field as "Note ID" might be unusual but it's acceptable in Anki. So maybe some time in the future some other person might face the same problem. If you can reproduce this you could include a warning in the manual or maybe something in the code?

Anyway, thanks again for this very useful piece of software.

**Comment by louietan** [comment]
Thanks for reporting this, I'll mention this in README.

**Comment by 517qf** [comment]
Thanks for your quick reply and help with all my issues.

I think the limitation is somewhat different (but don't rely too much on my text because I'm just a computer user).

I tried with different software that connects to AnkiConnect. I tried with the chrome extension [Anki Quick Adder](https://chrome.google.com/webstore/detail/anki-quick-adder/gpbcbbajoagdgnokieocaplbhkiidmmb) and a basic note type. If I leave the front empty in this chrome extension it doesn't work and shows an error message. My conclusion: AnkiConnect doesn't create a note if the first field is empty. I would characterize this limitations as follows

```
** Working with Anki add-ons

    This package does not work if the first heading = field is empty.
    You might run into this limitation if you use an add-on that
    automatically fills the first field (e.g. ~Add note id~).
```

Maybe this wording is more clear than yours in eab7d647
